src/styles/header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+  @include desktop {
+    max-width: 1280px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+
+  }
+
+  &__nav {
+    padding: 15px 0 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    @include tablet {
+      padding: 20px 0 30px;
+    }
+
+    @include desktop {
```

Comment:
The 'nav' element should not have a max-width. It's width will be determined by its containing element.